vslib/src/sai_vs_switch.cpp

```diff
@@ -14,6 +14,8 @@
 
 #include <thread>
 
+sai_object_id_t g_vs_switch_id = SAI_NULL_OBJECT_ID;
```

Comment:
this is not needed, vs_switch_id can be obtained from object_id and checked whether is in objects map

Reply:
We need the switch_id inside vs_set_port_attribute(). 
Unfortunately, only two arguments are passed to it, namely port_id and attr. 
We are not able to determine switch_id from any of these arguments. Hence, we need a global.
Please let me know your views.

Reply:
yes you are able to determine that even using sai_switch_id_query api

Reply:
Sure, required modifications are made.